### DISCUSSION TITLE
Error while doing a get call on the quorum blockchain network

### DISCUSSION THREAD

**Discussion by trinayanbhatt** [ConsenSys/quorum#1647] [General]
Hi team,

I'm getting this error while calling the get function of the deployed smart contract:

```
"error": {
        "code": -32000,
        "message": "execution reverted"
 }
```

The error isn't from the smart contract because the same contract is working fine on Remix and it seems like it is because of the gas limit or some quorum configuration. Can someone please help with the issue as I'm stuck with this error for some time now and I'm not able to resolve it.

**Comment by baptiste-b-pegasys** [comment]
Hello, can you enable the revert reason and see the description of it?
https://docs.goquorum.consensys.net/configure-and-manage/manage/revert-reason#enabling-revert-reason
